spec.html

```diff
@@ -14026,9 +14044,21 @@ <h2>Syntax</h2>
       <emu-note>
         <p>For example: `3in` is an error and not the two input elements `3` and `in`.</p>
       </emu-note>
-      <p>A conforming implementation, when processing strict mode code, must not extend, as described in <emu-xref href="#sec-additional-syntax-numeric-literals"></emu-xref>, the syntax of |NumericLiteral| to include <emu-xref href="#prod-annexB-LegacyOctalIntegerLiteral"></emu-xref>, nor extend the syntax of |DecimalIntegerLiteral| to include <emu-xref href="#prod-annexB-NonOctalDecimalIntegerLiteral"></emu-xref>.</p>
 
-      <emu-clause id="sec-static-semantics-mv">
+      <emu-clause id="sec-numeric-literals-early-errors">
+        <h1>Static Semantics: Early Errors</h1>
+        <emu-grammar>
+          NumericLiteral :: LegacyOctalIntegerLiteral
+
+          DecimalIntegerLiteral :: NonOctalDecimalIntegerLiteral
+        </emu-grammar>
+        <ul>
+          <li>It is a Syntax Error if the source code matching this production is strict mode code.</li>
+        </ul>
+        <emu-note>In non-strict code, this syntax is legacy.</emu-note>
```

Comment:
```suggestion
        <emu-note>In non-strict code, this syntax is Legacy.</emu-note>
```
since i think we want it to auto-link to the dfn added in #2125?

Reply:
Done.